src/Classes/DBPopulator.php

```diff
@@ -0,0 +1,92 @@
+<?php
+namespace ListingsApp\Classes;
+use PDO;
+class DBPopulator {
+    private ListingAPI $listingAPI;
+    private PDO $dbConnection;
+    private array $listings;
```

Comment:
Also worth making these protected so they can be inherited if needed by another classs